chat/message/message.go

```diff
@@ -202,38 +202,70 @@ func (m PrivateMsg) String() string {
 	return m.Render(nil)
 }
 
+var _ Message = &SystemMsg{}
```

Comment:
Can we make this change and the change to /list a separate PR? That's the only part I'm concerned about, the rest I'm happy to merge.

Reply:
reverted, will create a new PR